collections/Douce/MS_Douce_367.xml

```diff
@@ -37,45 +40,121 @@
+                     <decoNote type="flourInit"> One 9-line puzzle initial in red and blue with purple-red penwork including a human head in profile, at the beginning of the Fourth Age (fol. Cij recto).</decoNote>
+                     <decoNote type="flourInit">One 7-line initial in blue with purple-red penwork including a human head in profile, at the beginning of the Fifth Age (fol. Cxxx recto).</decoNote>
+                     <decoNote type="colInit">2- and 3-line initials throughout, alternately red or blue.</decoNote>
+                     <decoNote type="unfilled">A space was left on fol. ix verso for a map, never added. </decoNote>
                   </decoDesc>
+                  <bindingDesc>
+                     <binding notAfter="1500" notBefore="1459">
+                        <p>15th-century cuir ciselé. Sewn on four double bands laced into wood boards, covered with polished brown leather with cut designs against stippled backgrounds, the front cover with a central panel depicting a man and woman (a marriage proposal?) based on an engraving by Master E.S., approached by a jester(?) with pointed hat and shoes; below them one young man instructing or rebuking another, set against a background of scrolling foliage, and surrounded by panels of foliage and hybrid creatures; the back cover with, in the centre, a man-headed griffon-like creature above a lion, surrounded by foliate panels; each cover with four brass corner pieces, a centre-piece missing from the back cover; metal clasp fittings (the straps missing).</p><p>
+                           <bibl>F. A. Schmidt-Künsemüller, <title>Corpus der gotischen Lederschnitteinbände aus dem deutschen Sprachgebiet</title>, Denkmäler der Buchkunst, 4 (Stuttgart, 1980), p. 39 no. 234</bibl>; <bibl>L. Ubl, ‘Bodleian Library, MS. Douce 367: Von Kupferstichen und Einbänden – Die Excerpta Chronicarum’, <title>Oxford German Studies</title>, 46 (2017), 241–47.</bibl></p>
+                     </binding>
+                  </bindingDesc>
                </physDesc>
                <history>
                   <origin>
-                     <origDate calendar="Gregorian" when="1459">1459</origDate>
+                     <origDate calendar="Gregorian" notBefore="1459" notAfter="1500">15th century, second half, (soon?) after 1459</origDate>
                      <origPlace cert="low">
                         <country key="place_7000084">German</country>, <settlement key="place_7004334">Nuremburg</settlement> (?)<!--ORIGINAL: German, Nuremburg (?)--></origPlace>
                   </origin>
-                  <provenance notAfter="1834" notBefore="1757" resp="#MMM"><persName key="person_69848690" role="fmo">Francis Douce, 1757–1834</persName></provenance>
-                  <acquisition resp="#MMM" when="1834">Bequeathed to the Bodleian in 1834</acquisition>
+                  <provenance>1459 is the date of the text, not necessarily of this copy (<foreign>pace</foreign> Watson).</provenance>
```

Comment:
Move this into `<origin>` and give the reference for the inscription referred to?